packages/core/src/components/popover-menu/test/icons-fluid/popover-menu.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/popover-menu/test/icons-fluid/index.html';
+
+test.describe('tds-popover-menu-icons-fluid', () => {
+  test('renders icons-fluid popover-menu correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const triggerButton = page.getByRole('button').filter({ has: page.getByRole('img') });
+    await triggerButton.click();
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('menu item "The menu with adjusts to the widest word" exists', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const tdsMenuItemListLongText = page.getByText('The menu width adjusts to the widest word');
+    await expect(tdsMenuItemListLongText).toHaveCount(1);
+    await expect(tdsMenuItemListLongText).toBeHidden();
```

Comment:
Why is "toBeHidden" here?

Reply:
Without clicking the button to open the menu it is hidden so I also make sure that its hidden when I haven't opened it. I guess it does not reflect the test exactly tho. It says "exists" so

Reply:
OK, I get you - you are checking element in the DOM, but you do not really care if it up in front of the eyes or not, which also is true if user did not clicked on button. Makes sense, all good.